code/game/objects/items/explosives/bombvest.dm

```diff
@@ -18,14 +18,21 @@
 	if(activator.wear_suit != src)
 		to_chat(activator, "Due to the rigging of this device, it can only be detonated while worn.") //If you are going to use this, you have to accept death. No armor allowed.
 		return FALSE
+	if(!do_after(user, 3, TRUE, src, BUSY_ICON_DANGER, ignore_turf_checks = TRUE))
+		return FALSE
 	if(bomb_message) //Checks for a non null bomb message.
 		activator.say("[bomb_message]!!")
 		message_admins("[activator] has detonated an explosive vest with the warcry \"[bomb_message]\".") //Incase disputes show up about marines killing themselves and others.
 		log_game("[activator] has detonated an explosive vest with the warcry \"[bomb_message].\"")
 	else
 		message_admins("[activator] has detonated an explosive vest with no warcry.")
 		log_game("[activator] has detonated an explosive vest with no warcry.")
-	explosion(loc, 0, 2, 6, 5, 5) 
+
+	for(var/datum/limb/E in activator.limbs) //Oops we blew all our limbs off
```

Comment:
Shouldnt this use AS?

Reply:
Also why is the limb var named "E" sounds

Reply:
Copied from /mob/living/carbon/human/gib() in code\modules\mob\living\carbon\human\death.dm

Reply:
Since the list has known content I suppose it should. Done.